src/blocks/Slider/Slider.tsx

```diff
@@ -224,24 +224,24 @@ export const SliderBlock = (props: WithChildren<SliderProps>) => {
 
         return (
             slidesCountByBreakpoint > 1 && (
-                <div
+                <li
```

Comment:
Maybe wrapper of this element is semantically not `<ul>`?
I saw there only dots and arrows

Reply:
I see that dots in ul